index.bs

```diff
@@ -536,6 +536,20 @@ i.e. consistency within CSS.
 
 There is also a separate section on <a href="#naming-consistency">naming consistency</a>.
 
+<h3 id="specs-include-guidance">Follow guidance from feature specifications</h3>
+
+The specifications for individual features
+should include guidance for using those features
+consistently across other specifications.
+When using a feature,
+look for that guidance and follow it.
+An incomplete list of such guidance follows:
```

Comment:
```suggestion
When using a feature,
follow guidance in the specification for that feature.

Specifications for features
that will be used by other specifications
should include guidance for using those features.
This ensures that the feature is used correctly
and consistently
throughout the platform.

An incomplete list of such guidance follows:
```

Reply:
I strongly prefer this version over the original PR.